### ISSUE TITLE
KCHS

### ISSUE THREAD

**Issue by wkitty42** [terrasync/main#374]
i'm only doing the commercial parking and taxiways... KCHS is a dual purpose airport serving also as an USAF base as well as a Boeing maint/build site...

- created the 15 parking places around concourses A and B at the commercial terminal
- restricted concourse A to Delta
- restricted concourse B to American (AAL), JetBlue (JBU), and United (UAL)...
SouthWest (SWA) also flies here but is not in the list of available airlines currently using KCHS so cannot add them to the restriction list yet
- created basic taxiway paths for commercial craft to reach either end of either runway
- requested the removal of two shared object craft that were blocking parking A1 and A4

i'm not planning on doing the military parking or taxiways and i do not mind if someone else takes this ground net and enhances it with them...

aside: i know that parking can be limited to certain airline codes... can taxiways also be restricted in the same way?

**Comment by wkitty42** [comment]
i'm going to push the button on this one and see how it works to get into the stream and be distributed...

it may not be pretty but it works :smile_cat:

**Comment by wkitty42** [comment]
hope i didn't mess anything up by pushing the button too many times... the one under the wrench/spanner on the left menu doesn't seem to do anything... i ended up using the one on the top right when in view mode at the selected airport...

selecting the airport in 0.0.20 was a little tricky but 0.0.19 acted weird, too... under the wrench, i clicked on the center target on two of my airports... they didn't do anything... i had to search them, go into edit mode, save to get out, and then the target on the wrench tab would work to get to that airport... albeit a little off-center of the airport but it still got me where i needed to be ;)

**Comment by wkitty42** [comment]
i must be doing something wrong or i've found a bug in FGA 0.0.20... i've still not seen a pull request for KCHS :cat2: 

i don't see anything in the console debug log stuff...

**Comment by wkitty42** [comment]
maybe just not clicking too soon is the answer? i see that KCHS now has a pending pull request... YAY! :smile_cat:

**Comment by sfrsfrsfr** [comment]
hi wkitty42, so can this issue be closed?

**Comment by wkitty42** [comment]
no, i've not finished it yet... still have more to do as well as simply waiting on FGA to flesh things out and settle down...
